spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -235,7 +235,8 @@ public abstract class CDI<T> implements Instance<T> {
 }
 ----
 
-A portable extension or other object may obtain a reference to the current container by calling `CDI.current()`. `CDI.getBeanManager()` may be called at any time after the container fires the `BeforeBeanDiscovery` container lifecycle event until the container fires the `BeforeShutdown` container lifecycle event.
+A portable extension or other object may obtain a reference to the current container by calling `CDI.current()`.
+`CDI.getBeanManager()` may be called at any time after the container fires the `BeforeBeanDiscovery` container lifecycle event until the container fires the `BeforeShutdown` container lifecycle event.
```

Comment:
This usage is misleading, as this is not a static method.  If we're saying that `CDI.current()` can blow up, then the case Mark described should never happen.